src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/interface.py

```diff
@@ -41,6 +43,8 @@
 # the image detector to be used
 detector_thread = None
 
+logger = logging.getLogger()
```

Comment:
Does logger have to be a global variable? I thought that only import logging was enough and then, for example, logging.debug("A frame recieved using ws")

Reply:
True, I will change it

Reply:
removed